### PR TITLE
ENH: Add figure 2b

### PR DESCRIPTION
**Pull request by ElDeveloper**

### CODE REVIEW

File: notebooks/fig2b.ipynb [hunk 1]
```diff
@@ -0,0 +1,191 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Global Foodomics Visualization\n",
+    "\n",
+    "This figure re-visualizes the food metabolome dataset used in [Tripathi and Vazquez-Baeza et al. 2020](https://www.biorxiv.org/content/10.1101/2020.05.04.077636v1). This is an untargetted metabolomics study looking food-associated samples in the context of a cladogram created using [*q2-qemistree*](https://github.com/biocore/q2-qemistree).\n",
```
Comment (fedarko) [inline notebooks/fig2b.ipynb]:
```suggestion
    "This figure re-visualizes the food metabolome dataset used in [Tripathi and Vazquez-Baeza et al. 2020](https://www.biorxiv.org/content/10.1101/2020.05.04.077636v1). This is an untargeted metabolomics study looking at food-associated samples in the context of a cladogram created using [*q2-qemistree*](https://github.com/biocore/q2-qemistree).\n",
```

File: notebooks/fig2b.ipynb [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    "\n",
+    "The main goal of this figure is demonstrating the ease of visualizing feature prevalence on an EMPress tree, allowing us to do things like quickly see the relationship between clades and sample types.\n",
```
Comment (fedarko) [inline notebooks/fig2b.ipynb]:
```suggestion
    "The main goal of this figure is demonstrating the ease of visualizing feature prevalence on an EMPress tree, allowing us to do things like quickly see which types of samples contain features present in certain clades.\n",
```

subjective, feel free to ignore this one

File: notebooks/fig2b.ipynb [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+   "outputs": [],
+   "source": [
+    "observations = fm.query('superclass != \"SMILE parse error\" and superclass != \"unclassified\"').index.tolist()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Similarly, we'll also focus only on the sample with a *common meal type* annotation."
```
Comment (fedarko) [inline notebooks/fig2b.ipynb]:
```suggestion
    "Similarly, we'll also focus only on the samples with a *common meal type* annotation."
```

**Review by fedarko — CHANGES_REQUESTED** [review]
Thanks @ElDeveloper, this looks good to me. A few small wording suggestions, and it looks like this PR doesn't contain the QZV, but aside from that this should be good.

**Comment by ElDeveloper** [comment]
Thanks for the review @fedarko should be addressed 👍

**Comment by fedarko** [comment]
Sweet, thanks @ElDeveloper!

**Comment by ElDeveloper** [comment]
Thanks @fedarko

**Pull request merged**